tests/CNV-72329/implementation-python/conftest.py

```
@@ -0,0 +1,49 @@
+update testing scenarios.
+"""
+
+import pytest
+
+# Note: admin_client, unprivileged_client, and namespace fixtures
+# are provided by the main tests/conftest.py and should NOT be
+# redefined here.
+
+
+@pytest.fixture(scope="module")
+def skip_if_feature_gate_disabled(admin_client):
```

Ruff (0.15.0)

[warning] Unused function argument: `admin_client`

(ARG001)